SamplesDashboard/SamplesDashboard/ClientApp/src/components/layout/PageTitle.tsx

```diff
@@ -5,7 +5,8 @@ export default function PageTitle(props: any) {
 
     const classNames = mergeStyleSets({
         pageTitle: {
-            fontSize: FontSizes.large
+            fontSize: FontSizes.large,
+            padding: '5px'
         }
```

Comment:
move style definitions to their own files to make things more readable.

Reply:
Yeah, I will do so. Allow me to open this as a separate issue as it denotes a separate task.